<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListSiege</class>
 <widget class="QDialog" name="ListSiege">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAvion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>9</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>9</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capcit√©</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCapaciteAvion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>9</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>9</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID du vol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIDduVol">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>9</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>37</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>37</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>54</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="pbValiderSige">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAnnuler">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
